src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverter.cs

```diff
@@ -30,7 +30,7 @@ public abstract partial class JsonConverter
         /// <summary>
         /// Can the converter have $id metadata.
         /// </summary>
-        internal virtual bool CanHaveIdMetadata => true;
+        internal virtual bool CanHaveIdMetadata => false;
```

Comment:
What impact will this have for 3rd-party JsonConverter-derived types?

Reply:
None. Most converters including custom converters don't support emitting id metadata by design, so a `true` default does not reflect reality.